Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -380,6 +379,40 @@ script:
       name: org_address_line_one
       required: true
       type: textArea
+    - default: true
+      defaultValue: 'True'
+      description: Network Configuration - Dns Name Settings - Clone DNS Names
+      name: clone_dns_names
+      required: true
```

Comment:
You are right, some of the commands are not necessary to be required, but there was an issue when I changed them to non-required - as there are too many arguments for the command, those non-required arguments are not showing in the GUI task as an optional argument unless I change them to be required.  I am wondering if you know why this happens, and how we could get those non-required arguments showing in GUI task. Otherwise, we have to keep them as required.

Reply:
I understand the problem,
and we need to check why this is happening, and maybe open an issue for it to fix it.
but it is not possible to make arguments as required if they are not
because it will make it difficult for users to use them

Reply:
The suggested modifications have been accepted and updated. 
As we do not run Xsoar on a server, instead we are using cloud based Xsoar, we could not generate the Readme file.
Last time Shai helped us generated the Readme, and we provided the debug logs for the commands we added.
Could you help us do the same thing please? I will provide the debug logs for all the commands as below. Please let me know your thought. Thank you!

8 commands added. 
akamai-get-production-deployment,
akamai-get-change-history,
akamai-modify-appsec-config-selected-hosts,
akamai-patch-papi-property-rule-siteshield,
akamai-update-appsec-config-version-notes,
akamai-new-or-renew-match-target,
akamai-patch-papi-property-rule-generic,
akamai-get-papi-property-rule

Modified 7 commands:
create_enrollment  - adding two arguments,
update_property  - adding three arguments,
new_match_target  - adjusting the format,
activate_appsec_config_version  - adjusting the format,
clone_appsec_config_version  - adding parameters to api call,
patch_papi_property_rule_origin  - adjusting the format,
activate_papi_property - adding a argument

Reply:
Hi Moshe, I have collected the debug logs for 15 commands above and sent to your slack. Can you please check it? Thanks.   Also can you give me your email address, so we could connect easily.